pages_builder/generate_pages.py

```diff
@@ -87,7 +94,41 @@ def render_page(self, input_file):
                 partial['pageTitle'] +
                 " - Digital Marketplace frontend toolkit"
             )
-            if "examples" in partial:
+            if (
+                "examples" in partial
+            ):
+                template_file = (
```

Comment:
Be good to use [os.path.join](https://docs.python.org/2/library/os.path.html#os.path.join) to build this path.